priam/src/main/java/com/netflix/priam/services/BackupTTLService.java

```diff
@@ -87,7 +87,10 @@ public BackupTTLService(
     @Override
     public void execute() throws Exception {
         // Ensure that backup version 2.0 is actually enabled.
-        if (!backupRestoreConfig.enableV2Backups()) {
+        if (!backupRestoreConfig.enableV2Backups()
+                && backupRestoreConfig
+                        .getSnapshotMetaServiceCronExpression()
```

Comment:
If this is for the rollout period can we leave a comment indicating that?

Reply:
Irrespective of the rollout, no point running TTL service, if V2 backups are not enabled.